BLSQ/iaso
# IA-2699 Allow to limit authentication to third party authentication

Adding an environment variable DISABLE_PASSWORD_LOGINS that allows to disable logins using password (to only allow SSO )

Related JIRA tickets : IA-2699 (and sub stories)

I also removed a LOT of print() in the codebase so that the test outputs are clean. 

## Self proofreading checklist


## Doc
- under /docs

## How to test
Activate the environment variable. 

Go to /login  or /admin and check that no password field is available to fill
 /api/token/ should give you a 404 too

## Print screen / video

![CleanShot 2024-03-15 at 18 11 52@2x](https://github.com/BLSQ/iaso/assets/185797/826628f5-784f-42a5-b3f6-8a78103e96a8)
